### PR TITLE
BS-14324: Local environment configurations are needed when importing

### PR DESCRIPTION
**Pull request by apupier**
older workspace

### CODE REVIEW

**Comment by rbioteau** [comment]
PR updated

File: bundles/plugins/org.bonitasoft.studio.diagram.custom/src/org/bonitasoft/studio/diagram/custom/repository/ProcessConfigurationRepositoryStore.java [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /**
- * Copyright (C) 2012 BonitaSoft S.A.
- * BonitaSoft, 31 rue Gustave Eiffel - 38000 Grenoble
+ * Copyright (C) 2012-2015 Bonitasoft S.A.
```
Comment (apupier) [inline bundles/plugins/org.bonitasoft.studio.diagram.custom/src/org/bonitasoft/studio/diagram/custom/repository/ProcessConfigurationRepositoryStore.java]: this file is no more modified, so we should keep the headers as it was

File: bundles/plugins/org.bonitasoft.studio.common.repository/src/org/bonitasoft/studio/common/repository/Repository.java [hunk 2]
```diff
@@ -510,6 +511,16 @@ public void exportToArchive(final String fileName) {
         }
     }
 
+    private List<IResource> allRelatedResources(final IRepositoryStore<?> store) {
+        final List<IResource> realtedResources = new ArrayList<IResource>();
```
Comment (apupier) [inline bundles/plugins/org.bonitasoft.studio.common.repository/src/org/bonitasoft/studio/common/repository/Repository.java]: typo realtedResources --> relatedResources

**Comment by apupier** [comment]
Review done.
Waiting release 7.1.1 branch created to merge

**Pull request merged**